.pre-commit-config.yaml

```diff
@@ -1014,7 +1014,12 @@ repos:
         name: Update Airflow's meta-package pyproject.toml
         language: python
         entry: ./scripts/ci/pre_commit/update_airflow_pyproject_toml.py
-        files: ^.*/pyproject\.toml$|^scripts/ci/pre_commit/update_airflow_pyproject_toml\.py$
+        files: >
+          (?x)
+          ^.*/pyproject\.toml$|
+          ^scripts/ci/pre_commit/update_airflow_pyproject_toml\.py$|
+          ^providers/.*/pyproject\.toml$|
+          ^providers/.*/provider\.yaml$
```

Comment:
This wasn't present earlier?

Reply:
Yes. I found it as part of this PR, that we should trigger  the update when provider definiton change. Changing only provider.yaml files in the way that it would affect how airflow metadata project's pyproject.toml is defined (say Pyton 3.13 exclusion added) just failed static checks in the PR, where it runs them with `--all-files`